src/main/js/helpers/datetimeBuckets.js

```diff
@@ -204,9 +226,10 @@ const translateVGridDatetimeBuckets = (
  * @param {Array} values - list of all tick values.
  * @param {string} height - Y axis height.
  * @param {string} orientation - X axis orientation.
+ * @param {object} transit - gets transition based on pannig mode is enabled or not
  * @returns {object} - config with tick values and height.
  */
-const createDatetimeBucketConfig = (values, height, orientation) => ({
+const createDatetimeBucketConfig = (values, height, orientation, transit) => ({
```

Comment:
Avoid unnecessary abbreviation. 
`transition`

```js
height: `${height}`,
    settingsDictionary: {
        transition
    }
```

Reply:
I will correct it